templates/js/linker.v2.js

```diff
@@ -360,7 +362,7 @@
         setupPopup(options, mode);
 
         ns.elems = document.querySelectorAll(selector);
-
+        ns.quotationOnly = options.quotationOnly;
```

Comment:
Should also add `ns.parenthesesOnly = options.parentheses` to avoid accessing `parentheses` through `ns.link.arguments[0].parentheses` on line 386